components/AppBar/tests/unit/AppBarNavigationControllerAccessibilityPerformEscapeTests.swift

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+import MaterialComponents.MaterialAppBar
+
+class AppBarNavigationControllerAccessibilityPerformEscapeTests: XCTestCase {
```

Comment:
Should the class be prefixed? If so please update the file name as well.

Reply:
The class name is aligned with the existing convention in this directory, so keeping it as-is is within the scope of this PR.